### PR TITLE
feature/add-point-and-line-spacing-to-omp

### PR DESCRIPTION
**Pull request by michael-jaia**
## Summary
* Allows the operator to change the spacing between waypoints and lines when creating survey missions. Also removes the survey polygon code since it is no longer used and the code conflicted with the waypoint and line spacing.

## Details
* Creates separate functions for handling waypoint and line input changes. It is not ideal, but given the TypeScript type requirements I found it to be the cleanest solution.

## Testing
* Created survey missions with the differing waypoint and line values. Tested negative values to verify input is being handled correctly.

### CODE REVIEW

File: src/web/command_control/client/components/CommandControl.tsx [hunk 1]
```diff
@@ -1612,7 +1576,6 @@ export default class CommandControl extends React.Component {
 	 * Updates the mission layer features.
```
Comment (ferrom-jaia) [inline src/web/command_control/client/components/CommandControl.tsx]: Function Comment Needs Updating

File: src/web/command_control/client/components/MissionSettings.tsx [hunk 2]
```diff
@@ -20,9 +20,10 @@ export interface MissionParams {
 	missionType: 'editing' | 'polygon-grid' | 'lines' | 'exclusions'
 	numBots: number,
 	numGoals: number,
-	spacing: number,
-	orientation: number,
+    pointSpacing: number,
```
Comment (ferrom-jaia) [inline src/web/command_control/client/components/MissionSettings.tsx]: Spacing looks off here

File: src/web/command_control/client/components/MissionSettings.tsx [hunk 3]
```diff
@@ -208,15 +214,43 @@ export class MissionSettingsPanel extends React.Component {
...
-        const missionParams = this.state.missionParams
-        const target = evt.target as any
-        const key = target.name
-        const value = target.value as any
-
-        (missionParams as any)[key] = value
+    /**
+     * Prevents negative values or 0 from being used in data processing
+     * 
+     * @param {number} Value input value to be checked
```
Comment (ferrom-jaia) [inline src/web/command_control/client/components/MissionSettings.tsx]: Parameter name should be lowercase

File: src/web/command_control/client/components/MissionSettings.tsx (hunk 2 above)
Comment (michael-jaia) [reply src/web/command_control/client/components/MissionSettings.tsx]: I just re-indented the lines. Here's how they look in VS Code: 
image

**Review by ferrom-jaia — DISMISSED** [review]
Looks great, tested in sim. Please resolve merge conflicts.

**Review by ferrom-jaia — APPROVED** [review]
Looks good! Tested in sim.

**Pull request merged**